_posts/en/newsletters/2019-06-26-newsletter.md

```diff
@@ -0,0 +1,181 @@
+---
+title: 'Bitcoin Optech Newsletter #52'
+permalink: /en/newsletters/2019/06/26/
+name: 2019-06-26-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces a pending disclosure of minor
+vulnerabilities for older Bitcoin Core releases, suggests continued
+testing of RCs for LN software, and describes a proposed technique to
+make Bitcoin Core nodes a bit more resistant to eclipse attacks.  Also
+included are our regular sections on bech32 sending support and notable
+changes to popular Bitcoin infrastructure projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Update Bitcoin Core to at least 0.17.1:** Bitcoin Core contributor
+  Luke Dashjr [announced][dashjr vulnerabilities] a plan to disclose two
+  minor vulnerabilities for Bitcoin Core that affect older releases.
+  Neither vulnerability affects version 0.17.1 (released 25 December
+  2018) or later versions.  The vulnerabilities are expected to be
+  disclosed after August 1st.
+
```

Comment:
Do we want to recognize the contributor who raised notified the project about these issues?

Reply:
Feel free doing so!

You can include a link to https://twitter.com/practicalswift if anyone wants to follow up.

Also I suggest mentioning the assigned CVE:s (CVE-2017-18350, CVE-2018-20586) in the text.

Per the [project security policy](https://github.com/bitcoin/bitcoin/security/policy) I reported the full details to security@bitcoincore.org immediately after discovering the vulnerabilities. The issues were patched in a swift manner soon thereafter.

Luke helped me with all the external communication: he communicated with MITRE to get the CVE:s assigned and he also wrote the notice sent to the mailing list. He was very helpful in making sure this disclosure was handled in the most responsible way possible. Thanks Luke!

Reply:
Yes, it doesn't really make sense to name me here. @practicalswift should be attributed.